platform/ios/PolyPodApp/PolyPod/PolyStyle.swift

```diff
@@ -0,0 +1,74 @@
+import SwiftUI
+
+struct PolyStyle {
+    struct Spacing {
+        static let plSpace1x = 4.0
+        static let plSpace2x = {
+            2 * Self.plSpace1x
+        }()
+        static let plSpace3x = {
+            3 * Self.plSpace1x
+        }()
+        static let plSpace4x = {
+            4 * Self.plSpace1x
+        }()
+        static let plSpace5x = {
+            5 * Self.plSpace1x
+        }()
+        static let plSpace6x = {
+            6 * Self.plSpace1x
+        }()
+        static let plSpace7x = {
+            7 * Self.plSpace1x
+        }()
+        static let plSpace8x = {
+            8 * Self.plSpace1x
+        }()
+    }
+    
+    struct Radius {
+        static let plRadiusBase1x = 4.0
+        static let plRadiusBase2x = {
+            2 * Self.plRadiusBase1x
+        }()
+        static let plRadiusBase3x = {
+            3 * Self.plRadiusBase1x
+        }()
+        static let plRadiusBase4x = {
+            4 * Self.plRadiusBase1x
+        }()
+        static let plRadiusBase5x = {
+            5 * Self.plRadiusBase1x
+        }()
+        static let plRadiusBase6x = {
+            6 * Self.plRadiusBase1x
+        }()
+    }
+    
+    struct Font {
+        struct Family {
+            static let jostRegular = "Jost-Regular"
+            static let jostMedium = "Jost-Medium"
+        }
+        
+        struct Weight {
+            static let regular = SwiftUI.Font.Weight.regular
+            static let medium = SwiftUI.Font.Weight.medium
+        }
+        
+        struct Size {
+            static let xs = 12.0
```

Comment:
Are you using a grid here? Is there any relationship between these quantities? Can we use a base constant and compute the rest from there? Maybe base +2*x?

Reply:
We could. Just that the relationship between them was not defined in the figma file, as it was for Radius or Spacing for example. The idea was to basically have identical values here as in the figma file.

Reply:
Right, but you could still lave a 
```swift
static let grid = 2.0
```

And then 

```swift
             static let xs = 10.0 + grid*1;
```

For instance. Or even

```swift
static let base = 10
```